### ISSUE TITLE
Permanent Identifiers

### ISSUE THREAD

**Issue by dnes85** [AKSW/mexproject#47]
update the uris in order to support content negotiation
https://w3id.org/

**Comment by dnes85** [comment]
https://linkingresearch.wordpress.com/2016/01/17/permanent-identifiers-and-vocabulary-publication-purl-org-and-w3id/

**Comment by mommi84** [comment]
Agree, w3id.org is definitely the best choice now. At least, it is not read-only since last year...

**Comment by DiegoMoussallem** [comment]
I'm not confident about it, once it is not a pattern and only 34 people are inscribed in W3C (https://www.w3.org/community/perma-id/ ) in contrast of the author said in previous link something around 10k people. However, I liked it as an option for unavailability of Purl.org

**Comment by dnes85** [comment]
https://w3id.org/mex

**Comment by cirola2000** [comment]
Done.

**Comment by mommi84** [comment]
Beware that forwarding https://w3id.org/mex/ to http://aksw.github.io/mexproject/ means that you cannot use the w3id identifier for the MEX vocabulary (http://mex.aksw.org/), which is actually the intended use.

**Comment by dnes85** [comment]
That's true 👍  @cirola2000 could you please update the rules?

**Comment by dnes85** [comment]
folks, 
here you go:

1 - https://w3id.org/mex/$1 redirects to http://mex.aksw.org/$1 (AKSW proxy)
2 - http://mex.aksw.org/ redirects to http://aksw.github.io/mexproject/ (GitHub Project website) 

GitHub: last version of the vocabulary stays without versioning in the name, e.g.:
/mex-core https://raw.githubusercontent.com/AKSW/mexproject/master/vocabulary/mexcore.ttl
/mex-core https://raw.githubusercontent.com/AKSW/mexproject/master/vocabulary/mexperf.ttl
/mex-core https://raw.githubusercontent.com/AKSW/mexproject/master/vocabulary/mexalgo.ttl

We release each version by adding the version number, e.g.: https://raw.githubusercontent.com/AKSW/mexproject/master/vocabulary/mexcore1.0.1.ttl (it does not exist, just an example).

HTML documentation stays at dne5.com, once GitHub is not able to render HTML content.

**Comment by dnes85** [comment]
Also, @DiegoMoussallem has created the SPARQL link here 

http://mex.aksw.org/sparql
